youtube_dl_gui/locale/ar_SA/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -196,200 +191,187 @@ msgstr "إيقاف تشغيل النظام"
 
 #: youtube_dl_gui/mainframe.py:141
 msgid "Extension"
-msgstr ""
+msgstr "نوع الملف (الصيغة)"
 
 #: youtube_dl_gui/mainframe.py:142
 msgid "Size"
 msgstr "الحجم"
 
 #: youtube_dl_gui/mainframe.py:143
 msgid "Percent"
-msgstr "النِسبة"
+msgstr "النِسبة المئوية"
 
 #: youtube_dl_gui/mainframe.py:144
 msgid "ETA"
 msgstr "الوقت المتبقي"
 
 #: youtube_dl_gui/mainframe.py:145
 msgid "Speed"
-msgstr "السرعة"
+msgstr "سرعة التنزيل"
 
 #: youtube_dl_gui/mainframe.py:146
 msgid "Status"
 msgstr "الحالة"
 
 #: youtube_dl_gui/mainframe.py:235
 msgid "Get URL"
-msgstr ""
+msgstr "جد الرابط"
 
 #: youtube_dl_gui/mainframe.py:236
-#, fuzzy
 msgid "Get command"
-msgstr "أوامر"
+msgstr "جد أمر"
 
 #: youtube_dl_gui/mainframe.py:237
-#, fuzzy
 msgid "Open destination"
-msgstr "افتح وجهة المجلد"
+msgstr "افتح وجهة"
 
 #: youtube_dl_gui/mainframe.py:238
 msgid "Re-enter"
-msgstr ""
+msgstr "اعادة الادخال"
 
 #: youtube_dl_gui/mainframe.py:458
 msgid "Resume"
-msgstr ""
+msgstr "استئتاف"
 
 #: youtube_dl_gui/mainframe.py:480
 msgid "Video"
 msgstr "الفيديو"
 
 #: youtube_dl_gui/mainframe.py:484
 msgid "Audio"
-msgstr "الصوت"
+msgstr "ملف صوتي"
 
 #: youtube_dl_gui/mainframe.py:516
 msgid "No items selected. Please pick an action"
-msgstr ""
+msgstr "لم يتم تحديد اي عنصر"
```

Comment:
This translation seems incomplete since the "Please pick an action" part is missing